src/types/swaps.ts

```diff
@@ -1,25 +1,39 @@
 import { Address } from './address'
 
-export interface SwapTransaction {
-  swapType: 'same-chain' // only supporting same-chain swaps for now
+export type SwapTransaction =
+  | SameChainSwapTransaction
+  | CrossChainSwapTransaction
+
+interface SameChainSwapTransaction extends BaseSwapTransaction {
+  swapType: 'same-chain'
+}
+
+interface CrossChainSwapTransaction extends BaseSwapTransaction {
+  swapType: 'cross-chain'
+  // Swap duration estimation in seconds
+  estimatedDuration: number
+  maxCrossChainFee: string
+  estimatedCrossChainFee: string
+}
+
+interface BaseSwapTransaction {
+  swapType: 'same-chain' | 'cross-chain'
   chainId: number
   buyAmount: string
   sellAmount: string
-  buyTokenAddress: string
-  sellTokenAddress: string
-  // be careful -- price means different things when using sellAmount vs buyAmount
+  buyTokenAddress: Address
+  sellTokenAddress: Address
   price: string
```

Comment:
it was just a copy paste from another repo, it's not really relevant here since we don't use these values directly in this repo. I just replaced this with the one that's used in valora-rest-api here https://github.com/valora-inc/valora-rest-api/blob/main/src/swaps/providers/types.ts#L35